docs/sections/user_guide/cli/drivers/shave.rst

```diff
@@ -0,0 +1,93 @@
+``shave``
+=========
+
+The ``uw`` mode for configuring and running the UFS Utils preprocessing component ``shave``. Documentation for this UFS Utils component is :ufs-utils:`here <shave>`.
+
+.. code-block:: text
+
+   $ uw shave -h
```

Comment:
```suggestion
   $ uw shave --help
```
For consistency with most of the other docs. There are a couple of exceptions that I'm fixing on my branch.